addons/stock/models/stock_move.py

```diff
@@ -438,22 +438,20 @@ def _compute_json_forecast(self):
             forecast_lines = self.env['report.stock.report_product_product_replenishment']\
                 ._get_report_lines(None, product_variant_ids, wh_location_ids)
             for move in moves:
-                found = [l for l in forecast_lines if l["move_out"] == move and l["replenishment_filled"] is True]
-                if found:
-                    move_ins = list(filter(lambda report_line: report_line['move_in'], found))
-                    # If the move is linked to multiple ingoing moves, take only the last one to have the worst expected date.
-                    if len(move_ins):
-                        found = [move_ins[-1]]
-                    # The move is replenished but there's no expected date -> take from stock
-                    if found[0]["receipt_date_short"] is False:
+                lines = [l for l in forecast_lines if l["move_out"] == move and l["replenishment_filled"] is True]
+                if lines:
+                    move_ins_lines = list(filter(lambda report_line: report_line['move_in'], lines))
```

Comment:
Why those renames ? They are useless diff, isn't ?

Reply:
Because `lines` doesn't describe the same thing than `move_ins_lines`. Also, in my point of view, `found` is quite ambiguous.